docs/self-managed/optimize-deployment/setup.md

```diff
@@ -0,0 +1,62 @@
+---
+id: setup
+title: "Self-Managed setup"
+description: "Install and configure Optimize Self-Managed."
+---
+
+## Install using Docker
+
+The `camunda/optimize:latest` docker image can be used to run Optimize in Self-Managed as a container. Certain environment
+variables need to set for this to work correctly. See below for an example of how this could be done as
```

Comment:
is the be missing here?
```suggestion
variables need to be set for this to work correctly. See below for an example of how this could be done as
```